package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "reactjsdenver.com",
```

Comment:
I did a little searching on the web and it seems I can add that line anywhere in the package.json. I saw some doc that shows it above the license. Let me know if you prefer it on a different line.